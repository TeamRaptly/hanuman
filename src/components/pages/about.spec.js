import React from 'react';
import { shallow } from 'enzyme';
import About from './about';

describe('about page component', () => {
  it('renders without crashing', () => {
    const component = shallow(<About />);

    expect(component).toMatchSnapshot();
  });
});
