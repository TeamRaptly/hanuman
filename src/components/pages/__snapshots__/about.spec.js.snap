// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about page component renders without crashing 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Container))
    className="makeStyles-aboutpage-1"
  >
    <WithStyles(ForwardRef(Typography))>
      <HelmetWrapper
        defer={true}
        encodeSpecialCharacters={true}
      >
        <title>
          About page
        </title>
      </HelmetWrapper>
      <p>
        About Page
      </p>
      <Link
        to="/"
      >
        Home
      </Link>
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Container))>
  <MapWithSearch
    search={true}
  />
</Fragment>
`;
